packages/components/src/UserMenu/UserMenu.tsx

```diff
@@ -1,46 +1,51 @@
-import React from 'react';
-import styled from 'styled-components';
-import { border, color, layout, position, space } from 'styled-system';
+import React, { ReactNode } from 'react';
+import styled, { css } from 'styled-components';
+import {
+  border,
+  color,
+  layout,
+  position,
+  space,
+  typography,
+} from 'styled-system';
 
+import { T3nPro } from '@t3n/icons';
+import BookmarkBorder from '@t3n/icons/src/components/material/action/BookmarkBorder';
+import Person from '@t3n/icons/src/components/material/social/Person';
+import PersonOutline from '@t3n/icons/src/components/material/social/PersonOutline';
 import { ThemeProps } from '@t3n/theme';
 
-import Avatar from '../Avatar/Avatar';
 import Box from '../Box/Box';
-import Link from '../Link/Link';
-import Placeholder from '../Placeholder/Placeholder';
-import Text from '../Text/Text';
 
 export interface UserMenuProps {
-  loading: boolean;
-  loggedIn: boolean;
+  userEmail?: string;
   loginLink?: string;
   logoutLink?: string;
-  labelLink?: string;
+  proMembershipLink?: string;
+  readingListLink?: string;
+  accountLink?: string;
+  isProMember?: boolean;
   light?: boolean;
-
-  user?: {
-    label: string;
-    name: string;
-    avatarUrl: string;
-  };
-
-  itemGroups?: {
-    item: (JSX.Element | string)[];
-  }[];
+  userMenuItems?: ReactNode[];
```

Comment:
Hier würde es doch mehr Sinn machen, die Items einfach als Children zu übergeben, oder?

Reply:
Aber dann sehen die doch nicht so aus wie ich das gerne hätte 😅

Reply:
Wäre über die Children-API von React möglich, sehe aber auch gerade in der Doku, dass die nicht mehr empfohlen wird. Dann zumindest für die Komponente einfach in `items` umbenennen das Prop? `userMenu` ist ja redundant wenn die Komponente schon so heisst.

Reply:
Das mach ich :)